kolibri/core/assets/src/views/immersive-full-screen/index.vue

```diff
@@ -4,7 +4,14 @@
     <div class="top-bar">
       <router-link class="back-btn" :to="backPageLink">
         <mat-svg
-          class="back svg-back"
+          v-if="isRtl"
+          class="back back-btn-svg"
+          category="navigation"
+          name="arrow_forward"
+        />
+        <mat-svg
+          v-else
+          class="back back-btn-svg"
```

Comment:
It seems a little odd to switch between `arrow_forward` and `arrow_back` based on `isRtl` because it's always a "back" button. (Unlike below where the icons are named `keyboard_arrow_left` and `keyboard_arrow_right`, "back" and "forward" are semantic.)

Why not use your `rtl-icon` class above?

```html
<mat-svg
  category="navigation"
  name="arrow_back"
  class="back back-btn-svg"
  :class="{ 'rtl-icon': isRtl }"
/>
```

Reply:
actually [according to Material](https://google.github.io/material-design-icons/#icons-in-rtl), even the icons with "left" and "right" in the names should be mirrored. Seems like a good general strategy to use when possible?